test/cluster/mv/test_mv_building.py

```diff
@@ -113,3 +115,32 @@ async def test_view_building_with_tablet_move(manager: ManagerClient, build_mode
 
         for view in views:
             await wait_for_view(cql, view, len(servers))
+
+# While view building is in progress, drop the view and change the schema
+# of the base table. The view table's state may become inconsistent with
+# the base table because they are detached.
```

Comment:
Can you please confirm in the the commit message that this test really failed (at least sometimes) before this pull request, and passes afterwards?

Reply:
Also, shouldn't the comment above the test mention that you use an index, like you explained in the commit message?

Reply:
> Can you please confirm in the the commit message that this test really failed (at least sometimes) before this pull request, and passes afterwards?

I added to the commit message a part about this test failing before the patch and passing now

> Also, shouldn't the comment above the test mention that you use an index, like you explained in the commit message?

I changed the comment to refer to an index (like in the commit message)